web-src/src/components/PromptResultCard.js

```diff
@@ -100,6 +100,9 @@ const styles = {
     padding: 10px;
   `,
   variant: css`
+    display: -webkit-box;
```

Comment:
@askayastha22 Will it also work in Firefox?

Reply:
@vtsaplin Yes. I verified.

Reply:
https://experience-stage.adobe.com/?shell_ims=prod&ref=genai-plugin-dev&repo=wknd&owner=hlxsites&project=WKND#/@sitesinternal/aem/generate-variations/